.npmrc

```diff
@@ -1,2 +0,0 @@
-
-engine-strict=true
```

Comment:
This file was use to enforce the version of NodeJS we use for this project. Does it still force us to use the engine version specified in the package.json event without this file?

Reply:
Don't know but it seems really random to have a npmrc file when we don't use npm. There's probably a yarn option for that.

Reply:
Note: With yarn 2+, we have to install this plugin: https://github.com/devoto13/yarn-plugin-engines